### PR TITLE
removing filesES5,compressing all data folders in packages

### PR DESCRIPTION
**Pull request by Edebo**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to static food data?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] refactor
- [ ] documentation
- [ ] other

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

Are tests included?
- [ ] yes (*bugfixes and features will not be merged without tests*)
- [ ] no

Breaking Changes?
- [ ] yes (*breaking changes will not be merged unless absolutely necessary*)
- [ ] no

List any relevant issue numbers:

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Edebo** [comment]
after the whole changes i npm run build and am having this
missingshim

**Comment by atherdon** [comment]
ok, looks like it works, but with some warnings

File: package/babel.config.js [hunk 1]
```diff
@@ -19,22 +20,19 @@ const presets = [
...
       alias: {
-        '~': './src/data',
-        '@files': './src/files.js',
-        '@filesObjects': './src/filesObjects.js',
-        '@utils': './src/utils.js',
-        '@generator': './generator'
+        "~": "./src/data",
+        "@files": "./src/files.js",
+        "@utils": "./src/utils.js",
+        "@generator": "./generator"
```
Comment (atherdon) [inline package/babel.config.js]: can you remove this line - we don't need it anymore i think

**Comment by atherdon** [comment]
did you remove fileObjects from src?

File: package/babel.config.js (hunk 1 above)
Comment (Edebo) [reply package/babel.config.js]: this alias is used for our test

**Comment by Edebo** [comment]
> did you remove fileObjects from src?

i forgot to remove

**Pull request merged**